### PR TITLE
Added getLocalDiskFlag for SoftLayer_Virtual_Guest

### PR DESCRIPTION
**Pull request by mattcui**
@maximilien Need to add `getLocalDiskFlag` function to address [tracker 126844449](https://www.pivotaltracker.com/n/projects/1344876/stories/126844449), will submit another PR from `bosh-softlayer-cpi`. Please review and merge. Thanks.

### CODE REVIEW

File: services/softlayer_virtual_guest_test.go [hunk 1]
```diff
@@ -2127,4 +2127,41 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
...
 	})
+
+	Context("#GetLocalDiskFlag", func() {
+		BeforeEach(func() {
+			virtualGuest.Id = 1234567
+			fakeClient.FakeHttpClient.DoRawHttpRequestResponse, err = testhelpers.ReadJsonTestFixtures("services", "SoftLayer_Virtual_Guest_Service_getLocalDiskFlag.json")
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("local disk for virtual guest instance", func() {
```
Comment (maximilien) [inline services/softlayer_virtual_guest_test.go]: change description to: `It("finds local disk flag for local disk ...", ...`

File: services/softlayer_virtual_guest_test.go [hunk 2]
```diff
@@ -2127,4 +2127,41 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
 			})
 		})
 	})
+
+	Context("#GetLocalDiskFlag", func() {
+		BeforeEach(func() {
```
Comment (maximilien) [inline services/softlayer_virtual_guest_test.go]: Please add some error path tests. That is test to check when `false` is returned as well as `false` with error

**Comment by maximilien** [comment]
Please address feedback and add at least one integration test and I will merge. Thx

**Comment by mattcui** [comment]
@maximilien I already addressed your comments about unit and integration test, and passed the integration test through the dev branch pipeline. Please review again. Thanks.

**Comment by mattcui** [comment]
@maximilien this PR is a little urgent, I need to submit another one from `bosh-softlayer-cpi` after you merge it. And please help update Godeps of `bosh-softlayer-cpi` with this new change in `softlayer-go`. Thanks.

**Comment by maximilien** [comment]
OK merging. Best 👍

**Pull request merged**